### PR TITLE
Add an iv/cp calculator page 

### LINKED ISSUE #123: Add a "what level is this" page
A very lean page that just let you enter the IVs and CP of a pokemon, and told you what level it was, would be a nifty thing to have. No strong opinions about the UI, although I suppose if it was easy to use the same dropdowns and text input system as the rankings page (with a fourth field for CP), and let you enter multiple ones if you wanted to, that would make sense.

(IMHO something more complicated, like letting you optionally specify the current CP on the rankings page, would be harder to use; but if someone can come up with a clean interface for that, and it doesn't seem to clutter things up unnecessarily, then hey. But I kinda feel like a separate, super-simple page just for this would make the most sense.)

### PR DESCRIPTION
**Pull request by matthewyu01**
Added the basic functionality. I commented out the CDN and Google scripts since they kept on giving me errors, so you'll probably need to add them back. I also left a lot of code I copied over, but a lot of it can likely be taken out. And leaving the navbar alone for you to decide.

Closes #123

### CODE REVIEW

File: calculator.html [hunk 1]
```diff
@@ -0,0 +1,2178 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+  <!-- GA4 Tag -->
+  <!--<script async src="https://www.googletagmanager.com/gtag/js?id=G-LQY2YVNT21"></script><script>window.dataLayer=window.dataLayer || []; function gtag(){dataLayer.push(arguments);}gtag('js', new Date()); gtag('config', 'G-LQY2YVNT21');</script>-->
```
Comment (matthewyu01) [inline calculator.html]: Commented this out because of local errors but probably need to uncomment

File: calculator.html [hunk 2]
```diff
@@ -0,0 +1,2178 @@
...
+    }
+
+    p[id=IVinputShow],
+    small[id=IVinputHide] {
+      text-decoration: underline;
+      text-decoration-color: blue;
+    }
+  </style>
+  <!-- Begin Playwire Head -->
+  <!-- <script>window.ramp=window.ramp ||{}; window.ramp.que=window.ramp.que || []; </script> <script async src="//cdn.intergient.com/1025030/74482/ramp_config.js"></script> <script>window._pwGA4PageviewId=''.concat(Date.now()); window.dataLayer=window.dataLayer || []; window.gtag=window.gtag || function (){dataLayer.push(arguments);}; gtag('js', new Date()); gtag('config', 'G-JD7R9RWFZL',{'send_page_view': false}); gtag( 'event', 'ramp_js',{'send_to': 'G-JD7R9RWFZL', 'pageview_id': window._pwGA4PageviewId}); </script> -->
```
Comment (matthewyu01) [inline calculator.html]: Commented this out because of local errors but probably need to uncomment

File: calculator.html [hunk 3]
```diff
@@ -0,0 +1,2178 @@
...
+      }
+
+      function filterAdjusted() {
+        filterMod = true;
+        main();
+      }
+
+      function main(mon) {
+
+        return;
```
Comment (matthewyu01) [inline calculator.html]: main() and all calls to main() should probably be taken out as well as more functions. I didn't go through all the code I copied over but as far as I could tell, main() and many others aren't needed.

File: includes/calculateCP.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
...3301656, 0.654435634613037, 0.661219263506722, 0.667934000492096, 0.674581899290818, 0.681164920330047, 0.687684905887771, 0.694143652915954, 0.700542893277978, 0.706884205341339, 0.713169102333341, 0.719399094581604, 0.725575616972598, 0.731700003147125, 0.734741011137376, 0.737769484519958, 0.740785574597326, 0.743789434432983, 0.746781208702482, 0.749761044979095, 0.752729105305821, 0.75568550825119, 0.758630366519684, 0.761563837528228, 0.764486065255226, 0.767397165298461, 0.77029727397159, 0.77318650484085, 0.776064945942412, 0.778932750225067, 0.781790064808426, 0.784636974334716, 0.787473583646825, 0.790300011634826, 0.792803950958807, 0.795300006866455, 0.79780392148697, 0.800300002098083, 0.802803892322847, 0.805299997329711, 0.807803863460723, 0.81029999256134, 0.812803834895026, 0.815299987792968, 0.817803806620319, 0.820299983024597, 0.822803778631297, 0.825299978256225, 0.827803750922782, 0.830299973487854, 0.832803753381377, 0.835300028324127, 0.837803755931569, 0.840300023555755, 0.842803729034748, 0.845300018787384, 0.847803702398935, 0.850300014019012, 0.852803676019539, 0.85530000925064, 0.857803649892077, 0.860300004482269, 0.862803624012168, 0.865299999713897];
```
Comment (matthewyu01) [inline includes/calculateCP.js:2]: Copied over from [calculate.js](https://github.com/DeathbyToast/PvP_IVs/blob/9b045d11cb5af9ac12684fdb26035e47efd98686/includes/calculate.js#L2)

File: includes/calculateCP.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+function calculateCP(baseatk, basedef, basesta, atkIV, defIV, staIV, level) {
+  const index = Math.round((level - 1) * 2);
+  const cp = Math.max(10, Math.floor((baseatk + atkIV) * Math.sqrt(basedef + defIV) * Math.sqrt(basesta + staIV) * cpm[index] * cpm[index] / 10));
```
Comment (matthewyu01) [inline includes/calculateCP.js:6]: Copied over from [calculate.js L33](https://github.com/DeathbyToast/PvP_IVs/blob/9b045d11cb5af9ac12684fdb26035e47efd98686/includes/calculate.js#L33)

File: includes/calculateCP.test.js [hunk 6]
```diff
@@ -0,0 +1,101 @@
+// didn't import/export since modules don't work on some browsers
+// https://caniuse.com/es6-module
+function calculateCP(baseatk, basedef, basesta, atkIV, defIV, staIV, level) {
+  const index = Math.round((level - 1) * 2);
+  const cp = Math.max(10, Math.floor((baseatk + atkIV) * Math.sqrt(basedef + defIV) * Math.sqrt(basesta + staIV) * cpm[index] * cpm[index] / 10));
+  return cp;
+} 
+
+function testCalculateCP() {
+  const haunterCPs = [
```
Comment (matthewyu01) [inline includes/calculateCP.test.js:10]: Most of these values were gotten from in-game

File: calculator.html (hunk 1 above)
Comment (DeathbyToast) [reply calculator.html]: Yeah just the Google Analytics tags to track views and such. Shouldn't get in the way of the calculator, they're identical on all the pages.

File: calculator.html (hunk 2 above)
Comment (DeathbyToast) [reply calculator.html]: Yup this is the ad company's tags so they can track clicks / pay out for views on their ads

**Review by DeathbyToast — APPROVED** [review]

**Pull request merged**